pom.xml

```diff
@@ -60,6 +61,11 @@
             <version>${postgresql.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springdoc</groupId>
+            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
+            <version>${springdoc-openapi-starter-webmvc-ui}</version>
+        </dependency>
```

Comment:
**Suggestion:** It seems like the version of `springdoc-openapi-starter-webmvc-ui` is hardcoded in the dependency. It would be better to define it in the properties section for better maintainability. [maintainability]
```suggestion
        <properties>
            ...
            <springdoc-openapi-starter-webmvc-ui.version>2.3.0</springdoc-openapi-starter-webmvc-ui.version>
            ...
        </properties>
        ...
        <dependency>
            <groupId>org.springdoc</groupId>
            <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
            <version>${springdoc-openapi-starter-webmvc-ui.version}</version>
        </dependency>
```